tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -136,10 +133,12 @@ def main():
     # write one file per class provided
     for key in json_dict:
         out_file_name = f"{key}.json"
-        with open(out_file_name, 'w') as f:
-            logging.info(json.dumps(json_dict[key], indent=4))
-            json.dump(json_dict[key], f)
-
+        if path.exists(out_file_name):
+            rewrite = input(out_file_name + " already exists. Overwrite? (y/n)")
+            if rewrite == "y":
+                write_to_json(out_file_name,json_dict,key)
```

Comment:
more formatting notes: put a space after each comma in an argument list. i like to use [`autopep8`](https://pypi.org/project/autopep8/) to handle all the formatting for me, since it's tedious to do manually.

Reply:
wild to me that python doesn't have a simple core-library function for an "are you sure" prompt, too. i've looked into it as well, you're approaching this correctly.